db/seeds.rb

```diff
@@ -5,3 +5,1246 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+# Collection.create([{}, {}, {}])
+Collection.create([
+  {
+    name: "Spring",
+    location: "New York"
+  },
+  {
+    name: "Summer",
+    location: "New York"
+  },
+  {
+    name: "Autumn",
+    location: "New York"
+  },
+  {
+    name: "Winter",
+    location: "New York"
+  }
+])
+
+
+ClothType.create([
+  {
+    name: "dress"
+  },
+  {
+    name: "skirt"
+  },
+  {
+    name: "top"
+  },
+  {
+    name: "vast"
+  },
+  {
+    name: "coat"
+  },
+  {
+    name: "cape"
+  },
+  {
+    name: "pants"
+  }
+])
+
+apparel = Apparel.create(
+  #dd2_17
+  {  
+    name:"two-piece gather drape cape",
+    price:"HKD1,200",
```

Comment:
on store 1200 (no quotes, no comma) for the price. If you want to store the currency as well you should add a column to the apparels table